Sources/Sentry/SentryCrashReportConverter.m

```diff
@@ -268,10 +269,24 @@ - (SentryFrame *)stackFrameAtIndex:(NSInteger)frameIndex inThreadIndex:(NSIntege
     }
 
     NSMutableArray *frames = [NSMutableArray arrayWithCapacity:frameCount];
-    for (NSInteger i = frameCount - 1; i >= 0; i--) {
-        [frames addObject:[self stackFrameAtIndex:i inThreadIndex:threadIndex]];
+    SentryFrame *lastFrame = NULL;
+
+    for (NSInteger i = 0; i < frameCount; i++) {
```

Comment:
`m`: Is there a reason why we iterate now not in reverse order anymore? Below we call `reverseObjectEnumerator` so this seems like some unnecessary work.

Reply:
When we hit the marker, we mark the *previous* frame, thats why we gotta iterate from top frame down.

Reply:
You could also just mark the next frame when iterating reverse ;)